diagnostic.md

```diff
@@ -139,9 +167,8 @@ div {
 }
 ```
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+it would be positioned div2/div3/div4/div5/div1 assuming the could all fit on the same line. If they couldnt, it would look something like:  div2/div3/div4
+                      div5      div1
```

Comment:
.divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree)